lib/auth/auth_screen.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_chat/chat_list/chat_list_screen.dart';
+import 'package:keyboard_visibility/keyboard_visibility.dart';
+
+class AuthPage extends StatefulWidget {
+  @override
+  _AuthPageState createState() => _AuthPageState();
+}
+
+class _AuthPageState extends State<AuthPage> {
+  Alignment _alignment = Alignment.center;
+
+  @override
+  void initState() {
+    super.initState();
+    KeyboardVisibilityNotification().addNewListener(onChange: (bool isVisible) {
```

Comment:
bool is optional here, dart can understand the type of isVisible by onChange definition